templates/telemetry/downgrade/downgrade.4.parquetmr.txt

```diff
@@ -0,0 +1,14 @@
+message downgrade {
+  required binary id (UTF8);
+  required binary creationDate (UTF8);
+  @TELEMETRY_APPLICATION_1_TXT@
+  required binary clientId (UTF8);
+  required double version;
+  required group payload {
+    required binary lastVersion (UTF8);
```

Comment:
We generally recommend making fields `optional` unless the record is completely useless without them - that way if they are removed in later versions it won't cause trouble with the data storage.